tools/Data_explo/11_gala_autocorr.xml

```diff
@@ -0,0 +1,72 @@
+<tool id="tool_autocorr" name="Auto-correlation" version="0.0">

```

Comment:
I would recommend changing the folder name and the tool-id to something more meaningful.

Reply:
Version should also be changed. And please include `profile="20.01"`